packages/tools/src/eventListeners/touch/touchStartListener.ts

```diff
@@ -134,9 +134,9 @@ let tapState: ITouchTapListnenerState = JSON.parse(
 function triggerEventCallback(ele, name, eventDetail) {
   if (runtimeSettings.get('debug')) {
     if (name === 'CORNERSTONE_TOOLS_TOUCH_DRAG') {
-      console.debug(name);
```

Comment:
I'm not keen on having a log message here at all unless we can turn it on/off consistently - if it is ok with you, I will delete hte entire line.